### PR TITLE
Add option to disable automatic updates

### PR DESCRIPTION
**Pull request by flexyford**
This branch allows us to use an `{{each}}` inside `ember-impagination`. In the README I outline the flags that need to be set, and include some suggested methods.

Previously, we considered implementing a "setToEnd" function, which would be yielded on `records`. I do not think this is the API we want to reveal to users. Instead we should make them aware of [ember-in-viewport](https://github.com/dockyard/ember-in-viewport), and the [mut helper](https://github.com/emberjs/ember.js/blob/v2.2.0/packages/ember-htmlbars/lib/keywords/mut.js#L47).

### CODE REVIEW

**Comment by flexyford** [comment]
This branch also includes a fix to revert `ember-impagination`'s `virtual-each` dependency on version `0.0.14` which is what we need to make the Demo Application work. I will move this to a pull-request on master instead!

File: addon/components/impagination-dataset.js [hunk 1]
```diff
@@ -8,21 +8,25 @@ export default Ember.Component.extend({
...
 
   dataset: Ember.computed('page-size', 'load-horizon', 'unload-horizon', 'fetch', function() {
+    var round = Math.round;
     return new Dataset({
-      pageSize: this.get('page-size'),
-      loadHorizon: this.get('load-horizon'),
-      unloadHorizon: this.get('unload-horizon'),
+      pageSize: round(this.get('page-size')),
+      loadHorizon: round(this.get('load-horizon')),
+      unloadHorizon: round(this.get('unload-horizon')),
```
Comment (flexyford) [inline addon/components/impagination-dataset.js]: Will move `round`'s to different PR

File: addon/components/impagination-dataset.js [hunk 2]
```diff
@@ -45,7 +49,7 @@ export default Ember.Component.extend({
     this._super.apply(this, arguments);
 
     this.setInitialState();
-    this.get('dataset').setReadOffset(this.get('read-offset') || 0);
+    this.get('dataset').setReadOffset(Math.round(this.get('read-offset')) || 0);
```
Comment (flexyford) [inline addon/components/impagination-dataset.js]: Will move round's to different PR

File: package.json [hunk 3]
```diff
@@ -36,7 +36,7 @@
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.3",
     "ember-try": "0.0.6",
-    "virtual-each": "jasonmit/virtual-each#master"
+    "virtual-each": "0.0.14"
```
Comment (flexyford) [inline package.json]: Will move hard version dependency to different PR

**Comment by Robdel12** [comment]
How's this going?

**Comment by flexyford** [comment]
Closing in Favor of #130
